pkg/api/latest/latest_test.go

```diff
@@ -74,7 +74,7 @@ func TestRESTMapper(t *testing.T) {
 		t.Errorf("unexpected version mapping: %s %s %v", v, k, err)
 	}
 
-	if v, k, err := RESTMapper.VersionAndKindForResource("daemoncontrollers"); err != nil || v != "v1" || k != "DaemonController" {
+	if v, k, err := RESTMapper.VersionAndKindForResource("daemonControllers"); err != nil || v != "v1" || k != "DaemonController" {
```

Comment:
I'll change everything to "daemon" in the next PR, so it will be lower case. Leaving it for now, to avoid duplicate changes, but if you think I should still change it, I will!